website/src/release.md

```diff
@@ -0,0 +1,383 @@
+        - Check the header of the source code. This step needs docker to run.
+- Push the newly created branch to GitHub
+
+This script will create a new release under `dist`.
+
+For example:
+
+```shell
+> tree dist
+dist
+├── apache-iceberg-rust-0.2.0-src.tar.gz
+├── apache-iceberg-rust-0.2.0-src.tar.gz.asc
+└── apache-iceberg-rust-0.2.0-src.tar.gz.sha512
+```
+
+### Upload artifacts to the SVN dist repo
+
+SVN is required for this step.
+
+The svn repository of the dev branch is: <https://dist.apache.org/repos/dist/dev/iceberg/iceberg-rust>
+
+First, checkout Iceberg to local directory:
+
+```shell
+# As this step will copy all the versions, it will take some time. If the network is broken, please use svn cleanup to delete the lock before re-execute it.
+svn co https://dist.apache.org/repos/dist/dev/iceberg/iceberg-rust iceberg-dist-dev
+```
+
+Then, upload the artifacts:
+
+> The `${release_version}` here should be like `0.2.0-rc.1`
+
+```shell
+cd iceberg-dist-dev
+# create a directory named by version
+mkdir ${release_version}
+# copy source code and signature package to the versioned directory
+cp ${repo_dir}/dist/* ${release_version}/
+# check svn status
+svn status
+# add to svn
+svn add ${release_version}
+# check svn status
+svn status
+# commit to SVN remote server
+svn commit -m "Prepare for ${release_version}"
+```
+
+Visit <https://dist.apache.org/repos/dist/dev/iceberg/iceberg-rust/> to make sure the artifacts are uploaded correctly.
+
+### Rescue
+
+If you accidentally published wrong or unexpected artifacts, like wrong signature files, wrong sha256 files,
+please cancel the release for the current `release_version`,
+_increase th RC counting_ and re-initiate a release with the new `release_version`.
+And remember to delete the wrong artifacts from the SVN dist repo.
+
+## Voting
+
+Iceberg Community Vote should send email to: <dev@iceberg.apache.org>:
+
+Title:
+
+```
+[VOTE] Release Apache Iceberg Rust ${release_version} - Round 1
+```
+
+Content:
+
+```
+Hello, Apache Iceberg Rust Community,
+
+This is a call for a vote to release Apache Iceberg rust version ${iceberg_version}.
+
+The tag to be voted on is ${iceberg_version}.
+
+The release candidate:
+
+https://dist.apache.org/repos/dist/dev/iceberg/iceberg-rust/${release_version}/
+
+Keys to verify the release candidate:
+
+https://downloads.apache.org/iceberg/KEYS
+
+Git tag for the release:
+
+https://github.com/apache/iceberg-rust/releases/tag/${release_version}
+
+Please download, verify, and test.
+
+The VOTE will be open for at least 72 hours and until the necessary
+number of votes are reached.
+
+[ ] +1 approve
+[ ] +0 no opinion
+[ ] -1 disapprove with the reason
+
+To learn more about apache iceberg, please see https://rust.iceberg.apache.org/
+
+Checklist for reference:
+
+[ ] Download links are valid.
+[ ] Checksums and signatures.
+[ ] LICENSE/NOTICE files exist
+[ ] No unexpected binary files
+[ ] All source files have ASF headers
+[ ] Can compile from source
```

Comment:
Instead of having this in the mail, what do you think of having this just part of the docs: https://iceberg.apache.org/how-to-release/#validating-a-source-release-candidate

Reply:
I believe it's easier for PMC members to review releases if they have a checklist. But I'm fine if you want to in docs. Do you want to move this part?

Reply:
It is just a personal preference, it is not wrong having it here :)